frontend/src/lib/lemon-ui/LemonTree/LemonTree.tsx

```diff
@@ -293,7 +293,7 @@ const LemonTreeNode = forwardRef<HTMLDivElement, LemonTreeNodeProps>(
             <div className={cn('flex flex-col gap-y-px list-none m-0 p-0 h-full w-full', className)}>
```

Comment:
@adamleithp @mariusandra some slight changes here to core tree component. No strong opinions here/happy to take suggestions if there's a built in way to do what I'm doing here. The plus button on sources needs to be a direct link and not trigger a dropdown